documentation/docs/10-getting-started/10-introduction.md

```diff
@@ -10,10 +10,16 @@ title: Introduction
 
 ## What is SvelteKit?
 
-SvelteKit is a framework for rapid development of robust, performant web applications.
+SvelteKit is a framework for rapidly developing robust, performant web applications using [Svelte](https://svelte.dev/).
 
-Building an app with all the modern best practices is fiendishly complicated. Those practices include [build optimizations](https://vitejs.dev/guide/features.html#build-optimizations), so that you load only the minimal required code; [offline support](/docs/service-workers); [preloading](/docs/link-options#data-sveltekit-preload-data) pages before the user initiates navigation; [configurable rendering](/docs/page-options) that allows you to render your app [on the server](/docs/glossary#ssr) or [in the browser](/docs/glossary#csr-and-spa) at runtime or [at build-time](/docs/glossary#prerendering); and many other things. SvelteKit does all the boring stuff for you so that you can get on with the creative part.
+## What is Svelte?
 
-It uses [Vite](https://vitejs.dev/) with a [Svelte plugin](https://github.com/sveltejs/vite-plugin-svelte) to provide a lightning-fast and feature-rich development experience with [Hot Module Replacement (HMR)](https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/config.md#hot), where changes to your code are reflected in the browser instantly.
+You don't need to know Svelte to understand the rest of this guide, but it will help. In short, it's a way of writing user interface components — like a navigation bar, comment section, or contact form — that users see and interact with in their browsers. The Svelte compiler converts your components to JavaScript that can be run to render the HTML for the page and to CSS to style the page.
```

Comment:
I would move the first sentence to the end of this paragraph so that the question in the headline is answered directly. It would also give the opportunity to bring back the "see these links to learn more" sentence.

Reply:
done

I added back the link to the tutorial - one weird thing is that we link to learn.svelte.dev above and svelte.dev/tutorial here. Hopefully that will get resolved before too long as we finalize learn.svelte.dev though

I left out the link to the blog post as it's a bit of a weird place to point new users since a lot of the time is spent talking about the differences between Svelte 2 and Svelte 3, etc.